SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Utils/ApplicationLifecycleHelper.cs

```diff
@@ -104,8 +111,13 @@ public bool Enabled
             }
         }
 
+        public bool HasShownWindow => started && !suspended;
```

Comment:
Should only be mapped to `started` because the property is to indicate whether UI has been shown at any point in the past